app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +400,44 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div className={classes.menuContainer} ref={menuRef}>
+            <ReportButton />
+            <Button
+              className={classes.menuBtn}
+              onClick={() =>
+                setMenuOpen((prevMenuOpen: boolean) => !prevMenuOpen)
+              }
+            >
+              <MenuIcon />
+              <Avatar
+                user={user}
+                className={classes.avatar}
+                isProfileLink={false}
+              />
+            </Button>
+          </div>
+
+          <Menu
+            open={menuOpen}
+            anchorEl={menuRef.current}
+            onClose={() => setMenuOpen(false)}
+            classes={{
+              paper: classes.menu,
+            }}
+            anchorOrigin={{
+              vertical: "bottom",
+              horizontal: "right",
+            }}
+            getContentAnchorEl={null}
```

Comment:
Why is this prop necessary?

Also wondering if this is breaking arrow navigation between the menu items, as only the whole menu can be focused by keyboard at the moment. It would be nice and more accessible too to be able to go between the links.

Reply:
Ahhh never mind, looks like you get MUI warning without this while trying to set `anchorOrigin`